xplique/plots/metrics.py

```diff
@@ -0,0 +1,158 @@
+"""
+Plots for metric
+"""
+
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+
+from ..types import Optional, Dict, Any
+
+
+def barplot(
+        scores: Dict[str, Dict[str, float]],
+        sort_metric: str = None,
+        ascending: bool = False,
+        title: str = "",
+        filepath: str = None,
+        methods_colors: Optional[Dict[str, Any]] = None
+):
+    """
+    Make a bar-plot regrouping the score of each method for different metrics
+    Used to compare the different metric scores between methods
+
+    Parameters
+    ----------
+    scores
+        Dictionary of metrics scores
+        The keys are the metric names and the values also dictionaries where
+        The keys are the method names and the values the corresponding method-metric score.
+    sort_metric
+        Name of the metric used to sort the methods (a key of the directory)
+    ascending
+        Specify the sorting direction (need a sort_metric specified)
+    title
+        Title of the graphic
+    filepath
+        Path the file will be saved at.
+        if None, the function will call plt.show()
+    methods_colors
+        Dictionary of colors for each method,
+        the values should be supported by matplotlib as a color (named colors, RGB ...)
+
+    Returns
+    -------
+
+    """
+    # pylint: disable=invalid-name
+
+    metrics = list(scores.keys())
+    # sort values
+    if sort_metric is not None:
+        methods = [
+            k for k, _ in
+            sorted(scores[sort_metric].items(), key=lambda item: item[1], reverse=not ascending)
+        ]
+    else:
+        # get methods from a metric (order do not import)
+        methods = list(scores[list(scores.keys())[0]])
+
+    # set x abscesses for each metric
+    metrics_x = {metrics[i]: i for i in range(len(metrics))}
+
+    # set x delta for each method
+    width = (1-0.2) / len(methods)
+    # -0.4 to get 0.2 between each metric, 0.5 width to center the bars
+    methods_d = {methods[i]: -0.4+(i+0.5)*width for i in range(len(methods))}
+
+    # set methods colors
+    if methods_colors is None:
+        cmap = matplotlib.cm.get_cmap("Set3")
```

Comment:
Why Set3 in particular ? By default I would go with Set10 or Set20 to get better contrast as we often compare more than 3 metrics no ? :)

Reply:
In fact Set3 do not correspond to a set of 3 colors but to the third set. The colors are pastels, refer to the example image in the description of the PR to see those colors.
However I am open to suggestion to other sets, please select from https://matplotlib.org/stable/gallery/color/colormap_reference.html

Reply:
Oh you're right , my bad, I'm used to `set10` which is 10 colors ^^

Personnaly, I like to manipulate the colors for my plots, I would like to propose you to put the cmap name in parameters, to save you work I prepare a commit above yours that you can fixup with the modification (if you do not want it you can remove it)